.github/workflows/triage.yml

```
@@ -6,11 +6,10 @@ on:
       - opened
 
 env:
-  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GH_TOKEN: ${{ secrets.PROJECT_BOARD_AUTOMATION }}
   ORGANIZATION: ethereum
-  REPOSITORY: solidity
-  PROJECT_NUMBER: 43
-  DRY_RUN: false
+  PROJECT_NUMBER: 27
```

Please add the name of the board as a comment.

---

This looks like it's describing the field, not the value. This would be clearer IMO:
```suggestion
  PROJECT_NUMBER: 27 # Solidity Bug Triaging Board
```